pkg/third_party/retry/retry_test.go

```diff
@@ -159,8 +159,9 @@ func TestMaxDelay(t *testing.T) {
 	)
 	dur := time.Since(start)
 	assert.Error(t, err)
-	assert.True(t, dur > 170*time.Millisecond, "5 times with maximum delay retry is longer than 170ms")
-	assert.True(t, dur < 200*time.Millisecond, "5 times with maximum delay retry is shorter than 200ms")
+	fmt.Print(dur)
+	assert.True(t, dur > 50*time.Millisecond, "5 times with maximum delay retry is longer than 170ms", dur)
```

Comment:
why we change 170*time.Millisecond to 50*time.Millisecond?

Reply:
There are 5 times retry(without exponential backoff) with delay in range [10, 50], so that the expected should be in [10*5, 50*5].

Reply:
align with upstream.

https://github.com/avast/retry-go/blob/master/retry_test.go#L264